packages/typespec-python/src/emitter.ts

```diff
@@ -206,6 +220,19 @@ function emitBodyParameter(context: SdkContext, httpOperation: HttpOperation): B
     };
 }
 
+function isSubscriptionId(param: Record<string, any>): boolean {
+    return isArm && param.wireName === "subscriptionId";
+}
+
+function getDefaultApiVersionValue(context: SdkContext): string | undefined {
+    const defaultApiVersion = getDefaultApiVersion(context, getServiceNamespace(context));
+    if (!defaultApiVersion) {
+        const services = listServices(context.program);
+        return services.length > 0 ? services[0].version : undefined;
```

Comment:
`getDefaultApiVersion` can't find api-version for https://github.com/Azure/autorest.python/blob/fa2742f94e1dd724e64ff5832f690fdb7bf8a7d1/packages/typespec-python/test/specification/mgmt/sphere/main.tsp#L22-L26

In view of that we will migrate to `getAllOperations` soon, we add the logic here for now.

Reply:
Is it a TCGC bug and need to be fixed in `getAllOperations`?

Reply:
Yep